osfoffline/tasks/operations.py

```diff
@@ -280,12 +301,17 @@ class RemoteDelete(BaseOperation):
 
     def _run(self):
         resp = OSFClient().request('DELETE', self.remote.raw['links']['delete'])
-        assert resp.status_code == http.client.NO_CONTENT, resp
         db_model = Session().query(models.File).filter(models.File.id == self.remote.id).one()
-        DatabaseDelete(
-            OperationContext(db=db_model)
-        ).run()
-        Notification().info('Remote delete {}: {}'.format(db_model.kind.lower(), self.remote))
+        if resp.status_code == http.client.FORBIDDEN:
```

Comment:
There is still quite a lot of redundant code in these error messages. Can it be consolidated?